specification/botservice/resource-manager/Microsoft.BotService/preview/2021-05-01-preview/privatelinks.json

```diff
@@ -83,7 +83,7 @@
           "type": "string",
           "description": "The reason for approval/rejection of the connection."
         },
-        "actionRequired": {
+        "actionsRequired": {
```

Comment:
>actionsRequired [](http://example.com/codeflow?start=9&length=15)

How are multiple actions in the message separated? Generally ARM recommends multiple items are put into a collection so clients using the property don't need to do any additional parsing. Depending on how this property is used, it may not be too important, but worth considering.

Reply:
The property name is actually defined by the networking RP team, and it means to be a string that would be something like "add this private link in xyz". The fields here are directly taken from the NRP design spec, as is done by other RP's too (incidentally, i see that some RP's have it as actionRequired while others as actionsRequired, which is inconsistent and i wish these fields were in common-types for all RP's to refer and stay consistent about)

Reply:
Agree.